app_spec/cluster_test/index.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path')
+const { ConductorCluster } = require('../../hc_cluster_test')
+
+process.on('unhandledRejection', error => {
+  console.log('unhandledRejection', error.message);
+})
+
+const scenarioTest = async (numConductors = 2, debugging = false) => {
+  const dnaPath = path.join(__dirname, '..', 'dist/app_spec.dna.json')
+  const instanceId = 'test-1'
+
+  // just creates the instance
+  const cluster = new ConductorCluster(numConductors, { debugging })
+  // spawns the conductors and connects
+  // to their newly opened websocket connections
+  await cluster.initialize()
+  // install the DNA and create an instance
+  // with the test agent already in each Conductor
+  await cluster.batch(conductor => conductor.createDnaInstance(instanceId, dnaPath))
+
+  let enteringShutdown = false
+
+  // wait a maximum of 5 seconds for
+  // all the expected signals to arrive
+  // otherwise consider it a timed out failure
+  setTimeout(async () => {
+    if (!enteringShutdown) {
+      enteringShutdown = true
+      console.log('after 5 seconds, all nodes should be holding all entries and all links')
+      console.log(`There are only ${countHolding} after 5 seconds.`)
```

Comment:
```suggestion
      console.log(`There are ${countHolding} after 5 seconds.`)
```

There might be no holds barred, right?

Reply:
The idea of this is that if all HOLDs that were expected were found, it wouldn't be exiting with status 1

Reply:
Ah I missed the setTimeout